src/einsteinpy/symbolic/ricci.py

```diff
@@ -85,6 +87,60 @@ def from_metric(cls, metric):
         rc = ChristoffelSymbols.from_metric(metric)
         return cls.from_christoffels(rc)
 
+    def change_config(self, newconfig="uu"):
+        """
+        Changes the index configuration(contravariant/covariant)
+
+        Parameters
+        ----------
+        newconfig : str
+            Specify the new configuration. Defaults to 'uu'
+
+        Returns
+        -------
+        ~einsteinpy.symbolic.metric.RicciTensor
+            New Ricci with new configuration. Defaults to 'uu'
+
+        Raises
+        ------
+        ValueError
+            Raised when new configuration is not 'll' or 'uu'.
+            This constraint is in place because we are dealing with Ricci Tensor.
+
+        """
+        if newconfig == self.config:
+            return self
+        elif newconfig == "uu" or newconfig == "ll":
+            inv_met = RicciTensor(
+                sympy.simplify(sympy.Matrix(self.arr.tolist()).inv()).tolist(),
```

Comment:
Be *very* conservative about calling ``~sympy.simplify`` internally (in fact, it should most certainly be removed here). ``~sympy.simplify`` has an extremely high time complexity meaning that any moderately complex array will bring any script invoking this method to a halt.

Reply:
I'll keep it in mind, but I could not think of a simpler way to do it :/

Reply:
Anyways, this `change_config` codebase is gonna change, no worries!